### PR TITLE
[script.module.tzlocal] 2.0.0

### PR DESCRIPTION
**Pull request by SylvainCecchetto**
### Description
Python tzlocal library packed for Kodi.
tzlocal is a Python module that tries to figure out what your local timezone is.

Original python module: https://github.com/regebro/tzlocal
Forked Kodi module: https://github.com/jdollarKodi/script.module.tzlocal

This module will be a dependency of Catch-up TV & More video plugin (https://github.com/xbmc/repo-plugins/pull/2468).

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.tzlocal/.gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "lib"]
```
Comment (enen92) [inline script.module.tzlocal/.gitmodules]: remove

File: script.module.tzlocal/addon.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tzlocal"
+       name="tzlocal"
+       version="2.0.0"
+       provider-name="SylvainCecchetto,wwark">
```
Comment (enen92) [inline script.module.tzlocal/addon.xml]: provider name should be the module author, not yourself

Comment (enen92) [reply script.module.tzlocal/addon.xml]: Lennart Regebro

File: script.module.tzlocal/lib/CHANGES.txt [hunk 3]
```diff
@@ -0,0 +1,180 @@
+Changes
```
Comment (enen92) [inline script.module.tzlocal/lib/CHANGES.txt]: remove

File: script.module.tzlocal/lib/MANIFEST.in [hunk 4]
```diff
@@ -0,0 +1,5 @@
+include *.py
```
Comment (enen92) [inline script.module.tzlocal/lib/MANIFEST.in]: remove

File: script.module.tzlocal/lib/README.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
+tzlocal
```
Comment (enen92) [inline script.module.tzlocal/lib/README.rst]: remove

File: script.module.tzlocal/lib/setup.cfg [hunk 6]
```diff
@@ -0,0 +1,5 @@
+[check-manifest]
```
Comment (enen92) [inline script.module.tzlocal/lib/setup.cfg]: remove

File: script.module.tzlocal/lib/setup.py [hunk 7]
```diff
@@ -0,0 +1,43 @@
+from setuptools import setup, find_packages
```
Comment (enen92) [inline script.module.tzlocal/lib/setup.py]: remove

File: script.module.tzlocal/lib/spiny.conf [hunk 8]
```diff
@@ -0,0 +1,8 @@
+[spiny]
+environments=python2.5
+             python2.6
+             python2.7
+             python3.1
+             python3.2
+             python3.3
+             python3.4
```
Comment (enen92) [inline script.module.tzlocal/lib/spiny.conf]: lib/tests folder can be removed altogether

**Comment by Lunatixz** [comment]
https://github.com/xbmc/repo-scripts/pull/852 Thought we didn't need this module?

**Pull request merged**